ford/output.py

```diff
@@ -249,14 +249,14 @@ def writeout(self):
                 shutil.copy(src.path, os.path.join(out_dir, "src", src.name))
 
         if "mathjax_config" in self.data:
-            os.mkdir(os.path.join(out_dir, "js", "MathJax-config"))
+            mathjax_path = os.path.join(out_dir, "js", "MathJax-config")
+            if not os.path.isdir(mathjax_path):
+                os.mkdir(mathjax_path)
```

Comment:
Thanks @Meax5qiu ! A small suggestion:

```suggestion
            os.mkdirs(mathjax_path, exist_ok=True)
```

Reply:
Actually, I think I'd like to rewrite this whole function to use `pathlib.Path`, so I'll merge now. Thanks again for the bugfix!